Pynetbox_Data_Uploader/lib/utils/csv_to_dict.py

```diff
@@ -1,29 +1,75 @@
-from typing import List, Dict
-import pandas as pd
+from typing import Dict, List, Union, Optional
+from enums.dcim_device_no_id import DeviceInfoNoID
+from netbox_api.netbox_data import NetboxGetID
+from pandas import read_csv
 
 
-class CsvUtils:
+class FormatDict:
     """
-    This class provides methods to read data from csv files
-    and allow the data to be easily read and used elsewhere.
+    This class takes dictionaries with string values and changes those to ID values from Netbox.
     """
 
+    def __init__(self, netbox: Optional = None):
+        """
+        This method initialises the class with the following parameters.
+        Also, it allows dependency injection testing.
+        """
+        self.enums_no_id = DeviceInfoNoID
+        self.netbox = netbox
```

Comment:
Format dict shouldn't do anything which requires the netbox API....

Reply:
It doesn't do anything with Netbox. It just passes the api object through to the netbox_data.py . This allows you to initialise the api once at the start of a script and pass the api around from there, Rather than moving the url and tokens around and initialising a new api object for every method/class.

Reply:
We need to add a file called `__init__.py` to utils to resolve the import error, since this is a module but we're missing the module "marker file"

Reply:
utils does an `__init__.py` already in the directory.